assets/css/frontend/base/wordpress.css

```diff
@@ -0,0 +1,31 @@
+/* WordPress generated classes */
+
+.alignleft {
+	float: left;
+	margin-right: 1rem;
+}
+
+.alignright {
+	float: right;
+	margin-left: 1rem;
+}
+
+.aligncenter {
+	clear: both;
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+	text-align: center;
+}
+
+.wp-caption {
```

Comment:
Do we need `. wp-caption-text` here too? This is an awesome idea, though. I would love to have this be a part of the scaffold code.

Reply:
@dkoo left it out because I couldn't think of any properties for .wp-caption-text that is needed on all projects. Based on my experiences, only properties I add to .wp-caption-text are font related. And on some projects, it's not even included and inherits global styles. 

Any properties you add to .wp-caption-text on all WordPress projects? Feels like it can be left out.

Reply:
@smy315 @dkoo If we're going to do this let's remove our assumptions from the code and make sure everything is accounted for. If there's a class the editor generates, let's put it in the file.

Reply:
@dkoo take a look at my comment here? https://github.com/10up/theme-scaffold/pull/74#discussion_r223703028 thoughts?

Should I include .wp-caption-text commented out then?

Reply:
@smy315 I see what you mean and I understand why you only included the classes you did. It sounds like @timwright12 would prefer we include ALL classes generated by TinyMCE in the file but commented out if we don't need to declare any styles for them. For the scaffold, I would suggest we should only include the classes that we actually want to declare styles for, rather than including all classes. The presence of the wordpress.css file suggests that any overrides needed should go there. I think including all classes would make more sense if we do create an NPM package. I also think the NPM package idea is great, but that adding this file to the scaffold will help in the short-term while that package doesn't exist yet. What do you guys think?

Reply:
@smy315 @dkoo We don't need 100% coverage, but all the typographic and positioning basics should be in there.

A normal image in the classic editor outputs something like:

```
<div id="attachment_1668" style="width: 843px" class="wp-caption alignright"><img class="wp-image-1668 size-large" src="[imagepath]" alt="[alt]"><p class="wp-caption-text">Caption Here</p></div>
```
So we at least need these and their size/positioning variants:
`.wp-caption, .alignright, .size-large, .wp-caption-text`

`.size-large`, doesn't seem useful, so let's just add `.wp-caption-text` in there, then get this into the plugin repo so we can merge.